src/converter/FlameGraph.java

```diff
@@ -173,7 +177,9 @@ private String stripSuffix(String title) {
     }
 
     private int frameType(String title) {
-        if (title.endsWith("_[j]")) {
+        if (highlightStacksPattern.matcher(title).matches()) {
```

Comment:
I'm afraid, pattern matching on every single frame name may notably decrease conversion performance. At least, there should be a fast path to skip regex stuff when no `highlightStacksPattern` specified.